### ISSUE TITLE
Fix Maps (including sidebar filters, headers, text etc.), charts, and reports

### ISSUE THREAD

**Issue by robynlloos** [uno-isqa-8950/uno-cpi#549]
**Maps**
- Fix the map background so that it is more clearly a map – and not all gray (see the existing map background for an example – https://www.unomaha.edu/campus-commitment-to-community-engagement/partnerships.php) 
- Fix the gray boxes on all maps (will be based on the gray color that UCOMM provides)
- Remove secondary gray layered box behind map headings
- Remove effects placed around the buttons and boxes throughout the maps (all drop shadow effects should be removed)
- Buttons on maps should be consistent with button styles provided by UCOMM 
- Change the red button to display the filter for each map to be an arrow pointing left
- Make the map heading text smaller, with consistent font/text size and weight/thickness equal throughout
- Make the filter/entire sidebar background white, not gray
- Remove the heading “Filter” and push the below information up – no need for a filter heading there

**Charts and reports**
- Fix the gray boxes on all chart/report headings (will be based on the gray color that UCOMM provides)
- Remove effects placed around the buttons and boxes throughout the maps (all drop shadow effects should be removed)
- Buttons on maps should be consistent with button styles provided by UCOMM 
- Make the chart/report heading text smaller, with consistent font/text size and weight/thickness equal throughout
- Make the colors of the chart more sophisticated (see specifications about colors in the “Map Dots” section changes)

**Comment by caadler** [comment]
Maps have been converted to Google Maps and are substantially updated.  New functionality will need to be recorded in new tickets.
